indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -213,20 +216,30 @@ public Response apply(TaskQueue taskQueue)
               return Response.ok(ImmutableMap.of("task", task.getId())).build();
             }
             catch (EntryExistsException e) {
-              return Response.status(Response.Status.BAD_REQUEST)
-                             .entity(
-                                 ImmutableMap.of(
-                                     "error",
-                                     StringUtils.format("Task[%s] already exists!", task.getId())
-                                 )
-                             )
-                             .build();
+              return error(
+                  Response.Status.BAD_REQUEST,
+                  "Duplicate",
```

Comment:
On the query side the `error` key is meant to be "well known" error code: https://druid.apache.org/docs/latest/querying/querying.html#query-errors

Something like that on the Overlord side would be nice too. To encourage such standardization, how about creating an enum or a class-full-of-static-constants with all the possible error codes? That way, people updating or adding APIs won't be tempted to muck with the codes.